sql/core/src/main/scala/org/apache/spark/sql/execution/basicOperators.scala

```diff
@@ -307,6 +307,7 @@ case class MapPartitions[T, U](
     uEncoder: ExpressionEncoder[U],
     output: Seq[Attribute],
     child: SparkPlan) extends UnaryNode {
+  override def missingInput: AttributeSet = AttributeSet.empty
```

Comment:
I guess it is possible that generation of `SparkPlan` could introduce new `missingInput`. Having this will simplify the debugging?